realm/src/main/java/io/realm/internal/RealmCore.java

```diff
@@ -141,8 +141,8 @@ public static void loadLibrary() {
             else {
                 jnilib = "realm-jni-dbg";
             }
-            System.loadLibrary(jnilib);
         }
+        System.loadLibrary(jnilib);
```

Comment:
`loadLibraryWindows()` will load the library. This change implies that the library will be loaded twice on Windows (when we eventually support Windows).